ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRow.java

```diff
@@ -139,6 +149,14 @@ private PubsubMessageToTableRow(ValueProvider<String> tableSpecTemplate,
 
     final TableDestination tableDestination = new TableDestination(tableSpec, null,
         TIME_PARTITIONING);
+
+    if (tableRowFormat != null && tableRowFormat.get() == TableRowFormat.raw) {
+      TableRow tableRow = new TableRow();
+      AddMetadata.attributesToMetadataPayload(message.getAttributeMap()).forEach(tableRow::set);
+      tableRow.set("payload", message.getPayload());
```

Comment:
I'm pretty sure the payload bytes here are unzipped. We should zip before setting the payload field on the TableRow.

@scholtzan wrote a JS UDF to unzip bytes, so we should be able to peek at these zipped payloads in BigQuery. I'll follow up with her about getting that UDF integrated into bigquery-etl.

Reply:
See https://github.com/mozilla/bigquery-etl/issues/270

Reply:
we should set `--decompressInputPayloads=false` to preserve the compression we get from pubsub

Reply:
@whd ☝️

Reply:
ACK